ChangeLog

```diff
@@ -1,3 +1,11 @@
+2025-06-02  Kevin Ushey  <kevinushey@gmail.com>
+
+	* inst/include/Rcpp.h: Avoid copy when creating Language objects
+	* inst/include/Rcpp/Language.h: Idem
+	* inst/tinytest/cpp/language.cpp: Idem
+	* inst/tinytest/test_language.R: Idem
+	* inst/include/Rcpp/lgrow.h: Idem
```

Comment:
Maybe move this line up two and call the first 'tinytest' one 'new test'.  Not a biggie either way.